<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Livedash</class>
 <widget class="QMainWindow" name="Livedash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>691</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 213, 255, 255), stop:1 rgba(255, 0, 106, 255));
border-radius: 20px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButtonNepal">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>120</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Nepal</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonWHO">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>WHO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMoHP">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>MoHP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonBack">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonWorld">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>211</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>World</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
